### ISSUE TITLE
Update repository

### ISSUE THREAD

**Issue by sbagmeijer** [ulyaoth/repository#504]
Change repository to use domain "repos.ulyaoth.asia".

domain will now use cloudflare to speed up delivery.
move repository rpms to S3.
update nginx to redirect all urls to .asia (repository only) so cloudflare is used.

**Comment by sbagmeijer** [comment]
Step 1 is done now. https://repos.ulyaoth.asia is now live at cloudflare.
Step 2 is done now all rpms are now moved to s3.

**Comment by sbagmeijer** [comment]
New version is now available and everything has been moved correctly:
https://www.ulyaoth.net/resources/ulyaoth-repository.6/

**Comment by Hubbitus** [comment]
Unfortunately it does not works:
```
$ curl https://repos.ulyaoth.asia/centos/7/x86_64/os/repodata/repomd.xml
<html>
<head><title>404 Not Found</title></head>
<body>
<h1>404 Not Found</h1>
<ul>
<li>Code: NoSuchKey</li>
<li>Message: The specified key does not exist.</li>
<li>Key: centos/7/x86_64/os/repodata/repomd.xml</li>
<li>RequestId: AC0D24ACBEFE897B</li>
<li>HostId: IbtNkhpEbpHNeUnT8Po+QJZwHfN7LG9txGAbfUP3qJ96th6ZBtihQzMsJ4YylmpqnandA0zTuC0=</li>
</ul>
<hr/>
</body>
</html>
```

**Comment by sbagmeijer** [comment]
@Hubbitus 

I apologise when me did our move to S3 we made a mistake it is fixed now.

To explain all redhat clones use normally this kind of layout.

/redhat/7Server/
/oraclelinux/7Server/

So we had it setup as "/centos/7Server/":

But centos is using "/centos/7/" as variable.
CentOS 7:
{'arch': 'ia32e',
 'basearch': 'x86_64',
 'infra': 'stock',
 'releasever': '7',
 'uuid': 'ed7ed455-5a88-48e9-9928-6bc6c29c4237'}
 
 CentOS 6:
 {'arch': 'ia32e',
 'basearch': 'x86_64',
 'infra': 'stock',
 'releasever': '6',
 'uuid': '9469ca9c-48a0-4407-9ba2-13f4fd989856'}
 
 {'arch': 'i686',
 'basearch': 'i386',
 'infra': 'stock',
 'releasever': '6',
 'uuid': '2390ca3f-c43f-4de7-bacc-b81962f3068a'}


We have renamed the folder to correct it :/ I apologize for the inconvenience it was a big move last few days I hope it will all work fine again.
